buyer/issue/attach-document-api.md

```diff
@@ -137,15 +152,47 @@ Response status codes:
 
 #### Documents
 
-* `code` : optional document code or number. The document code should be unique within your company and immutable.
+* `code` : optional document code or number. The document code should be unique within your company and immutable. 
+
+{% hint style="info" %}
+If a document for an order(line) has the same code as an existing document at that order(line), the original will be overwritten. Otherwise, the newly attached document will be appended.
+{% endhint %}
+
 * `revision`: optional document revision \(or version\) code or number
 * `name` : optional document short \(file\) name
 * `description` : optional document description or long name
 * `type`: optional document business type
 * `objectId`: optional Tradecloud object identifier which was returned by the Tradecloud object-storage upload
 * `url`: optional document location if it is not stored in the Tradecloud object-storage.
 
-### Response
 
-When the `/api-connector/order/documents` API method returns HTTP status code 200, the order documents were successfully queued for processing by Tradecloud. Processing takes usually less then a second, after which the order documents are available in the portal and are forwarded to the supplier ERP integration.
+Body example:
 
+```text
```

Comment:
Isn't it better to point to the Swagger UI? That way we don't have to maintain this example in 2 places.

Reply:
Removed 2 example body's, added one (the other was already there) pointer to Swagger UI